lib/Agoneum/account/account.ex

```diff
@@ -63,7 +63,20 @@ defmodule Agoneum.Account do
   """
   def create_user(attrs \\ %{}) do
```

Comment:
@mbramson Is there a good reason to not just call this `create` since it will be invoked as `User.create()`?